app/src/main/java/com/example/notes2text/directoryJoin/usecases/JoinMenuInteractor.java

```diff
@@ -0,0 +1,50 @@
+package com.example.notes2text.directoryJoin.usecases;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.PopupMenu;
+import android.widget.Toast;
+
+import com.example.notes2text.adapters.ActivitySwitchController;
+import com.example.notes2text.directoryJoin.adapters.JoinController;
+import com.example.notes2text.usecases.FileMenuInteractor;
+
+import java.io.File;
+import java.util.ArrayList;
+
+public class JoinMenuInteractor extends FileMenuInteractor {
+
+    ArrayList<File> selectedList;
```

Comment:
consider making this variable private

Reply:
should be private by default, but will specify in pull request alongside documentation.